src/db/migrations/20250219182508_add_notifications_table.js

```diff
@@ -0,0 +1,32 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+export function up(knex) {
+  return knex.schema.createTable("notifications", (table) => {
+    table.increments("notification_id").primary();
+    table.string("label").nullable();
+    table.string("title").nullable();
+    table.string("description").nullable();
+    table.string("small_image_path").nullable();
+    table.string("big_image_path").nullable();
+    table.string("cta_label").nullable();
+    table.string("cta_link").nullable();
+    table.string("string").nullable();
+    table.string("audience").nullable();
```

Comment:
Could any of these be over 255 characters? I'd consider `text` instead of `string`, I believe knex defaults to a `varchar(255)` https://knexjs.org/guide/schema-builder.html#string which `text` is unlimited.

The distinction is a little confusing, in Postgres `varchar` and `text` are both unlimited by default, but you can specify a limit with `varchar(n)` which I suspect knex is doing from their docs.